README.md

```diff
@@ -8,14 +8,19 @@ A library of density matrix embedding theory (DMET) for lattice models and reali
 Installation
 ------------
 
-* Prerequisites
+* Prerequisites (will be automatically installed.)
```

Comment:
Thanks for your changes.
I would prefer to separate the PySCF out of the pip. As many users may have their local version of pyscf.
Maybe we can add one line in the README:
pip install pyscf

Reply:
I changed the description to:

`Prerequisites (will be automatically installed if not present.)`

If you already have pyscf installed,  either because you did `pip install pyscf` and installed from `PyPi` or because you did `pip install .` in the pyscf repo, then `pip` will not install `pyscf` for you.

Reply:
btw: may I suggest to do this PR https://github.com/gkclab/libdmet_preview/pull/20 first, it contains purely the support for pip. Just to disentangle the technical from the scientific pieces.